_config.yml

```diff
@@ -15,6 +16,7 @@ search_placeholder: "What are you looking for?"
 react-storybook: https://1-0-0-dev.dg2jutx9gylat.amplifyapp.com/
```

Comment:
not sure if this matters now cos later will change again. 

current react-storybook is https://2-0-0-dev.dg2jutx9gylat.amplifyapp.com/?path=/story/install--page

Reply:
lets update this once we confirm the domain names